### PR TITLE
Feature/bg 10

### PR DESCRIPTION
**Pull request by JustD1d1t**
Add timer that default has 10 minutes. If user type other time then it will be overwritten. It can be stopped when all ships are destroyed or when time has ended
Add finishGame function that is run when time has ended or someone destroyed all ships

### CODE REVIEW

File: src/game/game.html [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
+  </head>
 
-<body>
+  <body>
     <div>game</div>
-</body>
-
-</html>
\ No newline at end of file
+    <!-- paragraf z klasą timer na potrzeby wyświetlenia zegara. Do późniejszej zimany
```
Comment (ruljin) [inline src/game/game.html]: Typo - `zimany` 😄 ... besides I propose doing it everywhere the same way (TODO) and in English, in shortcut:
`//@TODO: Temporary paragraph for displaying timer`

File: src/game/gameTimer.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
+import { finishGame } from "./gameResult";
+
+const TIMER = document.querySelector(".timer"); // klasa kontenera w którym znajduje się zegar | do zmiany jeśli będziemy mieli widok
```
Comment (ruljin) [inline src/game/gameTimer.js]: `docs` or `todo` in English, please, and rather upon the line, you want to comment and not on its right or beneath it.

File: src/game/gameTimer.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    TIMER.textContent = `${minutes}:${seconds}`;
+
+    if (totalSeconds === 0) {
+      clearInterval(gameTime);
+      finishGame();
+    }
+  }, 1000);
+}
+
+export function stopTime() {
```
Comment (ruljin) [inline src/game/gameTimer.js]: `stopTimer` IMO.

File: src/game/gameTimer.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
+import { finishGame } from "./gameResult";
+
+const TIMER = document.querySelector(".timer"); // klasa kontenera w którym znajduje się zegar | do zmiany jeśli będziemy mieli widok
+
+let gameTime;
+
+export function setTime(time = 10) {
```
Comment (ruljin) [inline src/game/gameTimer.js]: `setTimer` IMO.

**Review by ruljin — CHANGES_REQUESTED** [review]

File: src/game/gameTimer.js (hunk 2 above)
Comment (JustD1d1t) [reply src/game/gameTimer.js]: forgot

File: src/game/gameResult.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
+import { displayModal } from "./gameOverModal";
+import { stopTime } from "./gameTimer";
+
+export function finishGame() {
+  let settlement;
+  let result;
+  const computerPoints = window.localStorage.getItem("computerPoints");
```
Comment (ruljin) [inline src/game/gameResult.js]: Use function from `utils` that @MaciekPon implemented.

File: src/game/gameTimer.js (hunk 3 above)
Comment (JustD1d1t) [reply src/game/gameTimer.js]: changed

File: src/game/gameResult.js [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+  const userPoints = window.localStorage.getItem("userPoints");
```
Comment (ruljin) [inline src/game/gameResult.js]: Use function from `utils` that @MaciekPon implemented.

**Review by ruljin — CHANGES_REQUESTED** [review]

File: src/game/gameTimer.js (hunk 4 above)
Comment (JustD1d1t) [reply src/game/gameTimer.js]: changed

File: src/game/gameResult.js (hunk 5 above)
Comment (JustD1d1t) [reply src/game/gameResult.js]: done

File: src/game/gameResult.js (hunk 6 above)
Comment (JustD1d1t) [reply src/game/gameResult.js]: done

File: src/game/game.html (hunk 1 above)
Comment (JustD1d1t) [reply src/game/game.html]: again 🤦

File: src/game/gameOverModal.scss [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+    @media screen and (min-width: $m-breakpoint) {
+      left: 45px;
+      right: 45px;
+    }
+  }
+  &__content {
+    max-width: 80vw;
+    max-height: 80vh;
+    z-index: 999;
+    h2 {
```
Comment (ruljin) [inline src/game/gameOverModal.scss]: I don't remember if I mentioned it here or not but NEVER don't style by `tags`. Only if it is required or the situation is good for it. Same thing for `h3`, and `a`.

**Review by ruljin — CHANGES_REQUESTED** [review]

File: src/game/gameTimer.js (hunk 2 above)
Comment (ruljin) [reply src/game/gameTimer.js]: Move it to `line 2` 😄

File: src/game/gameOverModal.scss (hunk 7 above)
Comment (JustD1d1t) [reply src/game/gameOverModal.scss]: I have add classes and styled using them

**Review by ruljin — APPROVED** [review]

**Pull request merged**